common/brave_features.h

```diff
@@ -17,7 +17,6 @@ COMPONENT_EXPORT(CHROME_FEATURES)
 extern const base::Feature kBraveRewards;
 #endif  // defined(OS_ANDROID)
 
-COMPONENT_EXPORT(CHROME_FEATURES)
```

Comment:
let's also remove this from the Android feature above. It does nothing, just adds confusion.